Packs/FeedCyrenThreatInDepth/Scripts/CyrenThreatInDepthRenderRelated/CyrenThreatInDepthRenderRelated.py

```diff
@@ -0,0 +1,124 @@
+
+
+def create_relationship_object(value: str, relationship_type: str, indicator_type: str,
+                               timestamp: str, entity_category: str, reputation: int = Common.DBotScore.NONE):
+    try:
+        timestamp_parsed = datetime.fromisoformat(timestamp.replace("Z", "+00:00"))
+        timestamp_human_readable = timestamp_parsed.strftime("%Y-%m-%d, %H:%M:%S")
+    except ValueError:
+        demisto.info("could not parse timestamp %s, keeping original string", timestamp)
+        timestamp_human_readable = timestamp
+
+    indicator_object = {
+        "Value": f"{value}\n",
+        "Reputation": SCORE_TO_REPUTATION_TEXT[reputation],
+        "Relationship Type": relationship_type,
+        "Indicator Type": indicator_type,
+        "Timestamp UTC": timestamp_human_readable,
+        "Entity Category": entity_category,
+    }
+
+    return indicator_object
+
+
+def cyren_feed_relationship(args) -> CommandResults:
+    if "indicator" not in args:
+        raise ValueError("Please provide 'indicator' argument!")
+
+    indicator = args["indicator"]
+    if isinstance(indicator, str):
+        try:
+            indicator = json.loads(indicator)
+        except json.JSONDecodeError:
+            raise ValueError("Please provide JSON-encoded 'indicator' param!")
+    elif not isinstance(indicator, dict):
+        raise ValueError("Please provide JSON-encoded 'indicator' param!")
+
+    headers = args.get("columns", ACCEPTED_HEADERS)
+    if isinstance(headers, str):
+        headers = [s.strip() for s in headers.split(",")]
+
+    check_acceptable_headers(headers)
+
+    relationships = indicator.get("CustomFields", {}).get("cyrenfeedrelationships", []) or []
+
+    content = []
+
+    for item in relationships:
+        ioc_value = item.get("value", "")
+        results = demisto.searchIndicators(value=ioc_value).get("iocs", [])
+
+        if results:
+            result = results[0]
+            ioc_score = result.get("score")
+            ioc_id = result.get("id")
+
+            content.append(create_relationship_object(
+                value=f"[{ioc_value}](#/indicator/{ioc_id})" if ioc_value else "",
+                relationship_type=item.get("relationshiptype"),
+                indicator_type=item.get("indicatortype"),
+                timestamp=item.get("timestamp"),
+                entity_category=item.get("entitycategory"),
+                reputation=ioc_score,
+            ))
+
+        else:
+            # In case that no related indicators were found, return the table without the link.
+            content.append(create_relationship_object(
+                value=ioc_value,
+                relationship_type=item.get("relationshiptype"),
+                indicator_type=item.get("indicatortype"),
+                timestamp=item.get("timestamp"),
+                entity_category=item.get("entitycategory"),
+            ))
+
+    output = tableToMarkdown("", content, headers, removeNull=True)
```

Comment:
Why there is no title to the table?

Reply:
We modeled this strictly after `Packs/CommonScripts/Scripts/FeedRelatedIndicatorsWidget` which also does not have a title. This is supposed to go in an indicator layout widget section and I think a title from the script output is not important there because you set in the layout UI.